src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/Questionnaire.kt

```diff
@@ -0,0 +1,82 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Column
+import jakarta.persistence.Convert
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.OneToMany
+import jakarta.persistence.SequenceGenerator
+import jakarta.persistence.Table
+import org.hibernate.Hibernate
+import org.hibernate.annotations.Generated
+import org.hibernate.type.YesNoConverter
+import org.springframework.data.annotation.LastModifiedBy
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+@Entity
+@Table(name = "QUESTIONNAIRES")
```

Comment:
also add `@EntityOpen`   which means we can create proxies and lazy load see CourtCase